Windows Community Toolkit.sln

```diff
@@ -804,6 +811,32 @@ Global
 		{75F9EE44-3EFA-47BC-AEDD-351B9834A0AF}.Release|x64.Build.0 = Release|Any CPU
 		{75F9EE44-3EFA-47BC-AEDD-351B9834A0AF}.Release|x86.ActiveCfg = Release|Any CPU
 		{75F9EE44-3EFA-47BC-AEDD-351B9834A0AF}.Release|x86.Build.0 = Release|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|ARM.Build.0 = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|ARM64.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|ARM64.Build.0 = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|x64.Build.0 = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Debug|x86.Build.0 = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|Any CPU.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|ARM.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|ARM64.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|x64.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|x86.ActiveCfg = Debug|Any CPU
+		{7878CD00-85E8-4D02-9757-8A43DB4C6510}.Native|x86.Build.0 = Debug|Any CPU
```

Comment:
Sorry, forgot this one!
```suggestion
```

Reply:
Ha, glad it was easy to spot for you! I'm so happy you're the ultimate CI/build guru here, because otherwise I don't know how long it'd have taken me to figure all of this out 😅